src/main/java/org/jabref/gui/ai/components/privacynotice/PrivacyNoticeComponent.java

```diff
@@ -100,4 +106,12 @@ private void openBrowser(String link) {
             dialogService.showErrorDialogAndWait(e);
         }
     }
+
+    @FXML
+    private void hideAITabs() {
+        EntryEditorPreferences entryEditorPreferences = preferences.getEntryEditorPreferences();
```

Comment:
@calixtus  do we need this restart?

Reply:
Live updated implemented in https://github.com/JabRef/jabref/pull/12958